GroundDBMemoryPool/src/include/rdma_server.hh

```diff
@@ -1,6 +1,6 @@
 #pragma once
 
-struct resources *init_server(
+void init_server(
```

Comment:
I don't think server needs struct resources*, since one side RDMA requires no further CPU after server is initialized. After `init_server()` , caller (memory pool) doesn't need this return value. In fact, I think caller should just go sleep, letting network card does remaining jobs. Need confirmation.

Reply:
I think we should keep it. The coordinator of the cluster would be put in the control memory node, so the resources will be used in future scenarios involving recovery, adding/removing compute nodes.

Reply:
Also, functions like allocating new page arrays on memory pool cannot be implemented by one-sided RDMA operations. Two-sided RDMA operations are necessary.